Makefile

```diff
@@ -31,15 +31,15 @@ testcoverage:
 installdeps:
 	pip install --upgrade pip
 	pip install -e .
-	pip install -r dev-requirements.txt
+	pip install -e .[dev]
```

Comment:
You should wrap these install commands in quotes so they will also work for people using zsh: `pip install -e ".[dev]"`.